src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -106,6 +106,7 @@
 				</content-container>
 			</div>
 		</article>
+		<what-is-kiva-modal v-if="kivaModuleExpEnabled && !hasEverLoggedIn" />
```

Comment:
With the way this is now this modal will be show for every visit to the borrower profile page until the visitor logs in. It should only be shown once. We should probably create a cookie after the modal has shown and only display the modal if the user does not have the cookie.